docs/data-management.rst

```diff
@@ -226,6 +232,87 @@ Typically an administrator works on behalf of other users in a group where the a
 
 #. When creating new Datasets or Projects during the move, note that these containers will belong to the owner of the data, not yourself. Also the links between the new containers and the moved data will belong to the owner of the data. This should help to facilitate a smooth workflow, retaining the data handling possibilities such as reorganizing the data, renaming the containers you created for them etc. for the owner of the data. 
 
+.. _Duplicatecli:
+
+*Command Line: Duplicating objects*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can duplicate objects in OMERO. The functionality is available only on the CLI for now. The duplication creates a full copy of the objects as if they were created independently. When desired, the newly duplicated objects can additionally be linked to other objects. When the duplicate is later deleted, it will not have an influence on the original, which stays preserved. Similarly, when the original is later deleted, the duplicate stays preserved as well.
+
+In case of Image objects, which have image files linked, the duplication creates a new image file which is linked to the original image file by a hard link. This means every duplication of an Image increases the number of hard links on the image file in Managed Repository of OMERO, but does not duplicate the image file itself, and thus does not increase the storage demands too much. 
+
+In case of File Attachment objects though, which also have files to them, each duplication will duplicate the linked file, thus doubling the storage space necessary for these File Attachment files.
+
+Resources
+---------
+
+-  Necessary software versions:
+
+   - **OMERO.server 5.6.3 or later**
+   - **omero-cli-duplicate plugin 0.4.0 or later**
+
+-  Documentation:
+
+   - `README of the omero-cli-duplicate repository <https://github.com/ome/omero-cli-duplicate/blob/master/README.rst>`_
+
+   - `Application Programming Interface documentation <https://docs.openmicroscopy.org/omero-blitz/5.5.7/slice2html/omero/cmd/Duplicate.html>`_
+
+-  Plugin for duplication on Command Line:
+
+   - `omero-cli-duplicate on PyPI <https://pypi.org/project/omero-cli-duplicate/#description>`_
+
+
+Setup
+-----
+
+**Duplicate plugin installation**
+
+- Go to the environment where you installed your OMERO.cli as specified under -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html.
+
+- Activate the virtual environment.
+
+- Run::
+    
+   $ pip install omero-cli-duplicate
+
+Step-by-Step
+------------
+
+#. On your local machine, open a terminal
+
+#. Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
+    
+   $ export PATH=/opt/omero/server/venv3/bin:$PATH
```

Comment:
Seems that the more extensive "Activate..." instructions should be in the Setup section, then in the Step-by-Step you can just have "Activate the virtual environment." since you're just repeating what you've already explained how to do?